Frends.PDF.Create/Frends.PDF.Create/Create.cs

```diff
@@ -0,0 +1,430 @@
+﻿using Frends.PDF.Create.Definitions;
+using MigraDoc.DocumentObjectModel;
+using MigraDoc.DocumentObjectModel.Shapes;
+using MigraDoc.DocumentObjectModel.Tables;
+using MigraDoc.Rendering;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Frends.PDF.Create
+{
+    /// <summary>
+    /// Task class.
+    /// </summary>
+    public class PDF
+    {
+        /// <summary>
+        /// Create PDF document from given content.
+        /// [Documentation] https://tasks.frends.com/tasks#frends-tasks/Frends.SFTP.WriteFile

```

Comment:
Update link and make it correct markdown (parethesis missing around link itself)